src/test/miner_tests.cpp

```diff
@@ -238,7 +238,7 @@ void MinerTestingSetup::TestPackageSelection(
     tx.vout[0].nValue = (100000000 - 10000) * SATOSHI;
     m_node.mempool->addUnchecked(entry.Fee(10000 * SATOSHI).FromTx(tx));
     pblocktemplate = AssemblerForTest(chainparams).CreateNewBlock(scriptPubKey);
-    BOOST_CHECK_EQUAL(pblocktemplate->block.vtx[4]->GetId(), lowFeeTxId2);
+    BOOST_CHECK_EQUAL(pblocktemplate->block.vtx[1]->GetId(), lowFeeTxId2);
```

Comment:
Why's the blocktime change the txn ordering?

Reply:
Will review

Reply:
Found it: https://github.com/EyeOfPython/cashd/blob/blocktime_2_minutes/src/test/miner_tests.cpp#L561

Here, the nSequence gets set to the MTP. But the MTP changes because of the faster blocktime. So the hash changes.

Expected behavior. Well spotted!